### PR TITLE
Produce events to error topic

### PR DESCRIPTION
**Pull request by Pchelolo**
If retries failed or message parsing failed - push a fatal error to the error topic.

The topic schema PR: https://gerrit.wikimedia.org/r/#/c/284986/

cc @wikimedia/services

### CODE REVIEW

File: lib/rule_executor.js [hunk 1]
```diff
@@ -67,9 +68,13 @@ class RuleExecutor {
 
     _safeParse(message) {
         try {
-            return JSON.parse(message);
+            return P.resolve(JSON.parse(message));
         } catch (e) {
-            this.log(`error/${this.rule.name}`, e);
+            return this.hyper.post({
```
Comment (gwicke) [inline lib/rule_executor.js]: Should we log these failures as well?

**Comment by gwicke** [comment]
LGTM, only one minor question inline.

**Comment by d00rman** [comment]
LGTM too, but we cannot merge this before https://gerrit.wikimedia.org/r/#/c/284986/ is resolved and merged.

**Comment by Pchelolo** [comment]
Also depends on #32 since we need to add `triggered_by` field to the error event

**Comment by ottomata** [comment]
Merged https://gerrit.wikimedia.org/r/#/c/284986/

**Pull request merged**